### ISSUE TITLE
Miners wondering with work to do.

### ISSUE THREAD

**Issue by LOL-Nation** [TangoTek/TekTopia-Community#488]
**Describe the bug**
In update 1.0.43 villagers wonder when they do not have work to do, but now miners wonder even with torches, food, pick and a lit mineshaft.

**To Reproduce**
1. Give a miner all necessary things to perform task,
2. observe that they sometimes mine and sometimes wonder.

**Expected behavior**
Only wonder when they can not mine.

** Server/Location **
What server were you playing on? Main(Survival) or Dev (Creative)?  Server 2
What are the world coordinates of your town hall marker? 4550 3570

**Comment by LOL-Nation** [comment]
This also happens with farms too.

**Comment by me2365** [comment]
Yes! They do this to me all the time. My mines are valid and all but they don't work.

**Comment by Grurim2k** [comment]
Yeah, Farmers, Miners, Lumberjacks, who all seemingly always have work to do, now enjoy leisurely strolls around the village most of the day heh

**Comment by JohnyDL** [comment]
Yeah I get this happen sometimes a farmer seems to path to a crop I've said ignore stands there and paths to another on their ignore until they randomly get a good crop

**Comment by MetriCfan** [comment]
My miners keep breaking by asking for as torch when they have crafted torches in inventory and have access to materials to make more. I have been resetting them to a different profession and then back to a miner and this works. However they just break again after a while. Their tunnels are being refreshed both by druids and myself, thus eliminating their existing placed torches. Could this be the issue.

**Comment by sbot50** [comment]
I had the same thing, then i notiched that my villager didn't claim a bed either. I killed the villager and spawned a new miner and she claimed the bed, still only wanders around tho.
Edit: tped her infront of the mine (with this command /tp @e[type=tektopia:miner] sbot50) she mined 2 blocks and then just stood there forever.

**Comment by owmee1** [comment]
it still wontwork
